lib/AST/GenericSignatureBuilder.cpp

```diff
@@ -1399,6 +1406,9 @@ bool PotentialArchetype::addConformance(ProtocolDecl *proto,
     return false;
   }
 
+  // Bump the generation.
```

Comment:
FWIW, I'm folding the `bumpGeneration()` change into my next attempt at this PR.